producer-model/src/main/scala/za/co/absa/spline/producer/model/v1_1/operations.scala

```diff
@@ -43,30 +45,32 @@ object OperationLike {
 
 case class DataOperation(
   override val id: Id,
+  override val name: Option[OperationLike.Name] = None,
   override val childIds: Seq[Id] = Nil,
-  override val output: Seq[Attribute.Id],
+  override val output: Seq[Attribute.Id] = Nil,
   override val params: Map[String, Any] = Map.empty,
   override val extra: Map[String, Any] = Map.empty
 ) extends OperationLike
 
 case class ReadOperation(
   inputSources: Seq[String],
   override val id: Id,
-  override val output: Seq[Attribute.Id],
+  override val name: Option[OperationLike.Name] = None,
+  override val output: Seq[Attribute.Id] = Nil,
   override val params: Map[String, Any] = Map.empty,
   override val extra: Map[String, Any] = Map.empty
 ) extends OperationLike {
-  override val childIds: Seq[Id] = Nil
+  override def childIds: Seq[Id] = Nil
 }
 
 case class WriteOperation(
   outputSource: String,
   append: Boolean,
   override val id: Id,
+  override val name: Option[OperationLike.Name] = None,
   override val childIds: Seq[Id],
   override val params: Map[String, Any] = Map.empty,
   override val extra: Map[String, Any] = Map.empty
 ) extends OperationLike {
-  override val output: Seq[Attribute.Id] = Nil
-
+  override def output: Seq[Attribute.Id] = Nil
```

Comment:
Same as above -- swagger generation tweaking. It's quite sensitive on conventions, so the way you define stuff in the classes matters.

Reply:
I think we'll need to switch to the Swagger definition first approach, and generate case classes from it. Otherwise it seems to be too fragile -- you add or change something in the code and don't realize that Swagger def is already different...